services/operator-value-parser.js

```diff
@@ -38,6 +38,10 @@ function OperatorValueParser() {
     } else if (value[0] === '*' && value[value.length - 1] === '*') {
       value = value.substring(1, value.length - 1);
       return { $like: '%' + value + '%' };
+    } else if (value[0] === '!' && value[1] === '*' &&
+      value[value.length - 1] === '*') {
+      value = value.substring(2, value.length - 1);
+      return { $notLike: '%' + value + '%' };
```

Comment:
I observe that the records having the filtered field set to `null` will be rejected by the query.
I think that they don't contain the `value` as they are `null` so they should be in the filtered subset of records. What do you think?